api/features/models.py

```diff
@@ -970,3 +979,52 @@ def get_update_log_message(self, history_instance) -> typing.Optional[str]:
 
     def _get_environment(self) -> typing.Optional["Environment"]:
         return self.feature_state.environment
+
+
+class FeatureExport(models.Model):
+    """
+    Stores the representation of an environment's export of
+    features between the request for the export and the
+    ultimate download. Records are deleted automatically after
+    a waiting period.
+    """
+
+    # The environment the export came from.
+    environment = models.ForeignKey(
+        "environments.Environment",
+        related_name="feature_imports",
+        on_delete=models.CASCADE,
+        swappable=True,
+    )
+
+    # This is a JSON string of data used for file download
+    # once the task has completed assembly.
+    data = models.CharField(max_length=MAX_FEATURE_EXPORT_SIZE)
+    created_date = models.DateTimeField("DateCreated", auto_now_add=True)
+
+
+class FeatureImport(models.Model):
+    """
+    Stores the representation of an environment's import of
+    features between upload of a previously exported featureset
+    and the processing of the import. Records are deleted
+    automatically after a waiting period.
+    """
+
+    # The environment the features are being imported to.
+    environment = models.ForeignKey(
+        "environments.Environment",
+        related_name="feature_exports",
+        on_delete=models.CASCADE,
+        swappable=True,
+    )
+    strategy = models.CharField(
+        choices=FEATURE_IMPORT_STRATEGIES,
+        max_length=50,
+        blank=False,
+        null=False,
+    )
+
+    # This is a JSON string of data generated by the export.
+    data = models.CharField(max_length=MAX_FEATURE_IMPORT_SIZE)
+    created_date = models.DateTimeField("DateCreated", auto_now_add=True)
```

Comment:
As above, I'd rather this was just `created_at = models.DateTimeField(auto_now_add=True)`